turk-one/app/Views-Controllers/Admin/admin-workers.html

```diff
@@ -190,11 +193,11 @@ <h6 class="bottom-buffer-25">References available</h6>
 
 				<p><span class="text-muted">explanation: </span> {{explain}}</p>
 
-				<p><span class="text-muted">time spent:</span> {{timeInMinutes}}m</p>
+				<p><span class="text-muted">time spent:</span> {{_time}}</p>
```

Comment:
Looks great, but why switch the naming to start with underscores? I know _propertyName is used elsewhere plenty, but the only place it shows up in Meteor/MongoDB is in the auto-generated _id field by Mongo. Maybe we can remove the underscore on these helpers to keep our naming scheme consistent for helpers? (camel-case, no underscore)